src/OpenApi/test/Microsoft.AspNetCore.OpenApi.SourceGenerators.Tests/SchemaTests.cs

```diff
@@ -58,6 +59,43 @@ public class BoardItem
     {
         public string Name { get; set; }
     }
+
+    /// <summary>
+    /// No XML comment processed here.
+    /// </summary>
+    private class Element
```

Comment:
Doesn't this type need to be used somewhere so we can check the lack of description?

Reply:
We have to rely on the check for compilation errors for this one. Since the type is private, we can't actually use it in the parameter list or return list for a method handler that is internal and/or public.